Gaijinizer patch/Map061.txt

```diff
@@ -138,48 +151,53 @@ No, I'm sure E-003 will get lost ....#MTLed
 ただでさえ劣勢な人間たちに、俺たちはみすみすヤバい脅威を
 新たに送り込むことになっちまうんだぜ…？
 > CONTEXT: Map061/events/5/pages/0/93/Dialogue/other2
-But what happens then? Even to the inferior
-people, we helplessly. What do you want to do?#MTLed
+But what happens then?
+
+At best, we've just knowingly sent in a dangerous
+new threat against humanity...#Ron
```

Comment:
Continuing from the previous set, we could alter the first line here to flow better after it:
>_**What would happen then?**_